docs: convert ECH Kibana settings page to use {settings} directives (#270144)
elastic/kibana: docs/reference/cloud/elastic-cloud-kibana-settings.md

Author: ```suggestion
applies_to:
  deployment:
    ech: ga
---
```

## Supported Kibana settings [ec-kibana-config]

> The following Kibana settings can be configured on {{ech}} deployments using the **User Settings** editor in the {{ech}} console. Settings not listed here are not supported on {{ech}}.

Author: ```suggestion
The following Kibana settings can be configured using the **Edit user settings** option when editing your {{ech}} deployment configuration. Settings not listed on this page are not supported on {{ech}}.
```
